vsintegration/src/FSharp.Editor/LanguageService/LegacyProjectWorkspaceMap.fs

```diff
@@ -20,33 +20,27 @@ open Microsoft.VisualStudio.LanguageServices.ProjectSystem
 open Microsoft.VisualStudio.Shell.Interop
 
 [<Sealed>]
-type internal LegacyProjectWorkspaceMap(workspace: VisualStudioWorkspaceImpl, 
+type internal LegacyProjectWorkspaceMap(workspace: VisualStudioWorkspaceImpl,
```

Comment:
What happens if this is just VisualStudioWorkspace (drop the Impl)? Are you still depending on anything in that anymore?

Reply:
I may need to update the package to get access to it.